x-pack/plugins/grokdebugger/public/sections/grokdebugger/components/pattern_input/pattern_input.js

```diff
@@ -28,11 +28,14 @@ app.directive('patternInput', function () {
         $scope.aceLoaded = (editor) => {
           this.editor = editor;
           editor.getSession().setUseWrapMode(true);
+          editor.renderer.setScrollMargin(7, 7, 0, 0);
           editor.setOptions({
             highlightActiveLine: false,
             highlightGutterLine: false,
             minLines: 3,
-            maxLines: 10
+            maxLines: 10,
+            autoScrollEditorIntoView: true,
+            mode: 'ace/mode/html'
```

Comment:
Why is HTML mode necessary? Since it seems unintuitive for this input, it might good to put a small explainer comment in the code.

Reply:
It's not, fresh eyes would have noticed this still being there. Thanks for picking this up.